backpack/extensions/secondorder/sqrt_ggn/base.py

```diff
@@ -0,0 +1,76 @@
+"""Contains base class for ``SqrtGGN{Exact, MC}`` module extensions."""
+from typing import Any, Callable, List, Tuple, Union
+
+from torch import Tensor
+from torch.nn import Module
+
+from backpack.core.derivatives.basederivatives import (
+    BaseDerivatives,
+    BaseParameterDerivatives,
+)
+from backpack.extensions.mat_to_mat_jac_base import MatToJacMat
+
+
+class SqrtGGNBaseModule(MatToJacMat):
+    """Base module extension for ``SqrtGGN{Exact, MC}``."""
+
+    def __init__(
+        self,
+        derivatives: Union[BaseParameterDerivatives, BaseDerivatives],
```

Comment:
```suggestion
        derivatives: BaseDerivatives,
```
`BaseParameterDerivatives` inherits `BaseDerivatives`. This is just a suggestion, not sure which one is better.

Reply:
Incorporated.